src/main/java/io/stargate/sgv2/jsonapi/StargateJsonApi.java

```diff
@@ -902,6 +902,24 @@
                                           }
                                       }
                                       """),
+              @ExampleObject(
+                  name = "createTextIndex",
+                  summary = "`createTextIndex` for text columns, in tables api",
+                  value =
+                      """
+                      {
+                             "createTextIndex": {
+                                 "name": "lexical_idx",
+                                 "definition": {
+                                     "column": "keywords",
```

Comment:
nit: is english better than "standard" which is the default ? and / or do we want to give an example of the good config with analyzers